src/ontology/mondo-edit.obo

```diff
@@ -213671,8 +213671,8 @@ xref: SCTID:198249000 {source="DOID:288"}
 xref: SCTID:237115002 {source="MONDO:equivalentTo", source="DOID:288"}
 xref: SCTID:76376003 {source="DOID:288"}
 xref: UMLS:C0341858 {source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="NCIT:C6996", source="OMIM:600458", source="DOID:288"}
-is_a: MONDO:0005133 {source="DOID:288", source="ICD10CM:N80.0", source="MONDO:Redundant"} ! endometriosis
-intersection_of: MONDO:0005133 ! endometriosis
+is_a: MONDO:0000931 {source="DOID:288", source="ICD10CM:N80.0", source="MONDO:Redundant"} ! endometrial disorder
```

Comment:
In Mondo, we want to keep the provenance of superclasses, so we change the superclass to an excluded superclass, rather than just deleting it. I'll take care of this. :)

Reply:
I changed the source for the superclass assertion to your ORCID @laurenechan